standard-cluster-services/src/main/java/org/terracotta/entity/IEntityMessenger.java

```diff
@@ -31,6 +31,12 @@
  */
 @CommonComponent
 public interface IEntityMessenger<M extends EntityMessage, R extends EntityResponse> {
+  /**
+   * Sends a message to delete the entity.  Must be called from the 
+   * {@link org.terracotta.entity.ActiveServerEntity#disconnected(org.terracotta.entity.ClientDescriptor) disconnected}
```

Comment:
Presumably we can also call from other places too?  What we're really saying is - destroy will fail if clients are still connected... it's just much more likely that no clients will be connected when calling from disconnected (when the last client gets disconnected).

Reply:
true

Reply:
It is actually guaranteed to fail if we call from anywhere else in the code due to the way we sequence a disconnect message.